content/docs/dvclive/index.md

```diff
@@ -0,0 +1,19 @@
+# dvclive
+
+[dvclive](https://cml.dev) is an open-source python library for monitoring
+metrics in machine learning projects. Use it to log metrics during training.
```

Comment:
```suggestion
[dvclive](https://cml.dev) is an open-source python library for monitoring
the progress of metrics during training of machine learning projects.
```
I'd suggest we emphasize that it's sort of a progress bar/tracker for your ML metrics (maybe even use that wording?), logging how metrics evolve while training iterations are still running, rather than at the end of training. @jorgeorpinel may have a better suggestion for wording.